src/components/DistanceRequest/index.tsx

```diff
@@ -186,8 +187,12 @@ function DistanceRequest({transactionID = '', report, transaction, route, isEdit
             // eslint-disable-next-line @typescript-eslint/naming-convention
             return {0: 'iou.error.atLeastTwoDifferentWaypoints'};
         }
-        return {};
```

Comment:
Sorry, but I just spotted this now. If this return statement is removed, then I don't think the `!isEmptyObject(getError())` check below is going to work. Does this method need to keep returning an empty object at the end of the `if` statements?

Reply:
Ah, OK. Thanks!